src/main/java/models/cell/TaxableCell.java

```diff
@@ -0,0 +1,70 @@
+package models.cell;
+
+import models.cell.ITaxable;
+
+public abstract  class TaxableCell implements ITaxable {
+
+    private int id;
```

Comment:
@ashudev6 These properties are already present inside Cell. Extend it. No need to repeat.